src/components/YieldPools/ListItem.tsx

```diff
@@ -74,7 +74,7 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
   const currentTimestamp = Math.floor(Date.now() / 1000)
 
   const qs = useParsedQueryString()
-  const tab = qs.tab || 'active'
+  const type = qs.type || 'active'
```

Comment:
For minimal change code :tien-sweating:
```suggestion
  const tab = qs.type || 'active'
```

Reply:
:)))))